nemo_skills/dataset/hmmt-feb-24/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
why do we need this?

Reply:
I'm assuming you're asking about the entire `__init__.py` file.  This is unrelated to this PR. Fixes the minor issues 
This was because while preparing the datasets, if you prepare things groupwise, [this line](https://github.com/NVIDIA/NeMo-Skills/blob/main/nemo_skills/dataset/prepare.py#L37) which checks for the `DATASET_GROUP` was not present for HMMT.

Reply:
oh wow, somehow i had the dataset in my branch. Weird. I removed the dataset. No worries!